### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by carloslfu**
## Description
Fix typo

### CODE REVIEW

**Review by Soean — APPROVED** [review]
Good catch, thanks for your contribution.

File: packages/keyboard-shortcuts/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Keyboard Shortcuts
 
-Keyboard shortcuts is a generic package that allow registering andd modifying shortcuts.
+Keyboard shortcuts is a generic package that allow registering and modifying shortcuts.
```
Comment (earnjam) [inline packages/keyboard-shortcuts/README.md:3]: While we're fixing that, there is a small subject-verb agreement error here too we should fix. 
```suggestion
Keyboard shortcuts is a generic package that allows registering and modifying shortcuts.
```

**Review by earnjam — CHANGES_REQUESTED** [review]
Thanks @carloslfu! I had one small suggestion in that sentence we should fix at the same time as this.

**Review by earnjam — APPROVED** [review]
Thanks!

**Pull request merged**